src/models/nebula49-1/assembly.md

```diff
@@ -318,17 +314,17 @@ The 140mm bottom case fan (`BQ SIW4-14025-LF-PWM`) is preinstalled, but can be r
 
```

Comment:
Step 4 above should be updated like nebula36 (no acrylic spacer, plus the new wording)